R/scan.R

```diff
@@ -0,0 +1,74 @@
+#' Read a scan results text file. scan.read returns a 1 segment dataframe.
+#' Reads in scan results and puts them in the same format as the output
+#' of dwt.read. See 'dwt', and 'segment' for more information.
+#' 
+#' Data is in seconds.
+#' @param fn, the file name to read from.
+#' @param separating_factor In lieu of a known time between segments, 
+#'        seperate with a multple of the longest dwell.
+#' @return A list of bursts.
+#' @examples
+#' \dontrun{
+#' seg <- scan.read('filename.txt')
+#'}
+#' @export
+#' @importFrom utils read.csv
+scan.read <- function(fn,separating_factor=1000){
```

Comment:
I would prefer "filename" as a variable to "fn". Just a tiny bit more obvious for people when they read the documentation